ocs_ci/utility/version.py

```diff
@@ -152,3 +153,115 @@ def compare_versions(expression):
         )
     v1, op, v2 = m.groups()
     return eval(f"get_semantic_version(v1, True){op}get_semantic_version(v2, True)")
+
+
+def get_dr_hub_operator_version(namespace="openshift"):
```

Comment:
is "openshift" namespace for DR ? don't make namespace as hardcode since we support multi namespace feature.

Reply:
is "openshift" namespace for DR ? -> DR operator applies on all namespaces, so I choose "openshift" ns

Reply:
could you keep ns as constant. All the NS are defined here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L251

Reply:
done